utils/mermaid.js

```diff
@@ -0,0 +1,192 @@
+const GRAPH_TITLE = new String("graph TB\n");
+const FULL_DEPTH = 64;
```

Comment:
Where does this come from?

Reply:
I updated to use the value in Javascript as HTML's initial value for "full".